airbyte-lib/airbyte_lib/executor.py

```diff
@@ -28,6 +28,10 @@ def ensure_installation(self):
     def install(self):
         pass
 
+    @abstractmethod
+    def get_tracking_information(self) -> Dict[str, str]:
```

Comment:
This might be semantic, but I slightly prefer a name like `get_telemetry_info` or `get_telemetry_markers` over `get_tracking_info`. The work 'tracking', to me at least, implies ability to personally identify or de-anonymize, which this isn't.

Reply:
Makes sense to me, I pondered the name a bit. Telemetry has the risk to be confused with telemetry in the sense of OpenTelemetry and the like, but it's probably better than tracking. "analytics" would be another option, but thinking about it calling this kind of thing telemetry is probably an industry standard for software running on the client.